### PR TITLE
Final integration

### PR DESCRIPTION
**Pull request by KaushikShivam**
Integrate logic with UI

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -5,16 +5,16 @@ const operate = (numberOne, numberTwo, operation) => {
   const numTwo = Big(numberTwo);
   switch (operation) {
     case '+':
-      return numOne.plus(numTwo);
+      return numOne.plus(numTwo).toString();
     case '-':
-      return numOne.minus(numTwo);
+      return numOne.minus(numTwo).toString();
     case '÷':
       if (numTwo === 0) return 0;
```
Comment (SunnyAsar) [inline src/logic/operate.js]: Hi, this is breaking the app `===` checks for type, don't forget button names are strings.

File: src/logic/operate.js [hunk 2]
```diff
@@ -5,16 +5,16 @@ const operate = (numberOne, numberTwo, operation) => {
...
-      return numOne.div(numTwo);
+      return numOne.div(numTwo).toString();
     case 'X':
-      return numOne.times(numTwo);
+      return numOne.times(numTwo).toString();
     case '%':
-      return numOne.mod(numTwo);
+      return numOne.mod(numTwo).toString();
```
Comment (SunnyAsar) [inline src/logic/operate.js]: for safety check the second number is not `'0'` here too.

File: src/logic/operate.js (hunk 1 above)
Comment (KaushikShivam) [reply src/logic/operate.js]: Done! 👍

File: src/logic/operate.js (hunk 2 above)
Comment (KaushikShivam) [reply src/logic/operate.js]: Done! 👍

**Comment by KaushikShivam** [comment]
Hello @SunnyAsar , I have made the necessary changes. :)

**Comment by KaushikShivam** [comment]
> Hi @KaushikShivam ,
> 
> Great job fixing the bugs.
> 
> Congrats, it is a
> 
> ### 👍
> You can go ahead and merge the project.

Thank you, @zenott

**Pull request merged**